public/manifest.webmanifest

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Hyperchannel",
+  "short_name": "Hyperchannel",
```

Comment:
@galfert Want to update this still?

Reply:
Right, forgot about that. Changed in 171cdf6.